backend/audit/intakelib/checks/check_cluster_names.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+
+from django.conf import settings
+from audit.intakelib.intermediate_representation import get_range_by_name
+from audit.intakelib.common import get_message, build_cell_error_tuple
+
+logger = logging.getLogger(__name__)
+
+
+def check_cluster_names(ir):
+    """Check that cluster names are valid when present."""
+
+    range_data = get_range_by_name(ir, "cluster_name")
+    errors = []
+    if range_data and ("values" in range_data):
+        valid_file = open(f"{settings.SCHEMA_BASE_DIR}/ClusterNames.json")
+        valid_json = json.load(valid_file)
```

Comment:
Catching the exception here, will mostly be to control the exception message otherwise we still want to pass this exception through since the goal is to validate intake and if we cannot validate then let user know that we are unable to validate